hw/bitstream/hyperdebug/BUILD

```diff
@@ -9,38 +9,23 @@ load("//rules:const.bzl", "KEY_AUTHENTICITY")
 package(default_visibility = ["//visibility:public"])
 
 filegroup(
-    name = "test_rom",
+    name = "bitstream",
     testonly = True,
     srcs = select({
-        "//hw/bitstream:bitstream_skip": ["//hw/bitstream:skip.bit"],
-        "//hw/bitstream:bitstream_vivado": ["//hw/bitstream/vivado:fpga_cw310_test_rom"],
-        "//hw/bitstream:bitstream_gcp_splice": [":gcp_spliced_test_rom"],
-        "//conditions:default": [":gcp_spliced_test_rom"],
+        "//hw/bitstream:bitstream_skip": ["skip.bit"],
+        "//hw/bitstream:bitstream_vivado": ["//hw/bitstream/vivado:fpga_cw310_test_rom_hyp"],
```

Comment:
Ouch, these were pretty broken before, haha. Lots of copy-paste bugs you're fixing. :)

Reply:
Yeah those are very wrong :)